### DISCUSSION TITLE
Why is the link to GitHub handle not fetched for one of the contributors?

### DISCUSSION THREAD

**Discussion by algomaster99** [jreleaser/jreleaser#1322] [Q&A] [answered]
Hi!

Thank you for the tool! I am using it to perform releases on [collector-sahab](https://github.com/ASSERT-KTH/collector-sahab). However, it does not fetch "Aman Sharma"'s github handle in the contributors. Could anyone point out why it may be happening?

**Comment by aalmiray** [comment, marked as answer]
JReleaser performs a user search based on email addresses found in commit messages. In your case it runs the following query https://github.com/search?q=mannu.poski10@gmail.com+in%3Aemail&type=Users which returns no matching results.

**Comment by algomaster99** [comment]
Thank you! The e-mail associated with my commits was not public. It should be fixed in the next release.
